### ISSUE TITLE
Probably shouldn't import tidyverse

### ISSUE THREAD

**Issue by RMHogervorst** [koaning/dumbr#2]
Great idea of the package! 
I noticed you import tidyverse, which is not recommended. You should probably import the individual packages.

**Comment by koaning** [comment]
yeah... this repo is a dummy repo. i tried getting travis to work. instead there's now a `helloworld` package. 

this is a package that id wanted to talk to you about. see slack ;)
